src/components/homePage.tsx

```diff
@@ -222,7 +226,12 @@ const HomePage = () => {
               </span>
             </div>
             <div className="flex items-center font-helvetica">
-              {width > 700 && <TimeTracker timeStamp={refreshTime} />}
+              {width > 700 && (
```

Comment:
I am not a fan of rendering an element and then hiding it. Ideally, we should either render or not-render.
Can you try something like this ? 
Untested code, Please test.
`{width > 700 && !showClass &&  <TimeTracker  timeStamp={refreshTime} />}`